crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -1940,10 +1941,10 @@ fn transitive_prerelease_and_stable_dependency_many_versions() {
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ Because only bluebird==1.0.0 is available and bluebird==1.0.0 depends on crow, we can conclude that all versions of bluebird depend on crow.
-          And because only crow<2.0.0b1 is available, we can conclude that all versions of bluebird depend on crow<2.0.0b1.
-          And because albatross==1.0.0 depends on crow>=2.0.0b1 and only albatross==1.0.0 is available, we can conclude that all versions of bluebird and all versions of albatross are incompatible.
-          And because you require bluebird and you require albatross, we can conclude that the requirements are unsatisfiable.
+      ╰─▶ Because only albatross==1.0.0 is available and albatross==1.0.0 depends on crow>=2.0.0b1, we can conclude that all versions of albatross depend on crow>=2.0.0b1.
+          And because only crow<2.0.0b1 is available, we can conclude that all versions of albatross depend on crow>3.0.0.
+          And because bluebird==1.0.0 depends on crow and only bluebird==1.0.0 is available, we can conclude that all versions of albatross and all versions of bluebird are incompatible.
+          And because you require albatross and you require bluebird, we can conclude that the requirements are unsatisfiable.
```

Comment:
I believe these are generally the result of dependencies being presented to PubGrub in a different order.

Reply:
Yeah it's a little weird but a known issue.